_posts/2019-09-06-kubernetes-and-hokusai.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: epic
+title: "Kubernetes and Hokusai"
+date: "2019-09-06"
+author: [matt-dole]
+categories: [artsy, beginners, engineering, hokusai, kubernetes, k8s]
+---
+
+When I joined Artsy Engineering a few months ago, I had roughly 0 knowledge of Kubernetes. I'd heard the term
+thrown around a few times, but had no idea how it worked or what it was good for.
+
+Artsy also has a super cool wrapper that we use to emulate a lot of Heroku CLI functionality for Kubernetes (and to
+do some other awesome stuff, read on for deets). It's fully open-source, and you should think about adopting it if
+you or your company uses Kubernetes.
+
+Read on for the full scoop.
+
+<!-- more -->
+
+# What is Kubernetes?
+
+If you already have a good idea of what Kubernetes is and how it works, feel free to skip this section. Since I
+still find Kubernetes pretty confusing, I thought I'd start by breaking down the basics.
+
+Kubernetes is a tool designed to _orchestrate containers at scale._
+
+Let's break that down a bit. First, some helpful vocab:
```

Comment:
Maybe collapse "break down" into just one occurrence or remove this sentence completely since it's mentioned just above.